git_commitai.py

```
@@ -78,16 +78,15 @@ def check_staged_changes(amend=False, auto_stage=False):
         # For --amend, we're modifying the last commit, so we don't need staged changes
         # But we should check if there's a previous commit to amend
         try:
-            run_command('git rev-parse HEAD', check=True)
+            run_command("git rev-parse HEAD", check=True)
             return True
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)